crates/apollo-parser/test_data/parser/err/0041_operation_definition_with_missing_selection_set.txt

```diff
@@ -1,10 +1,10 @@
-- DOCUMENT@0..19
-    - OPERATION_DEFINITION@0..19
+- DOCUMENT@0..16
+    - OPERATION_DEFINITION@0..16
         - OPERATION_TYPE@0..5
             - query_KW@0..5 "query"
         - WHITESPACE@5..6 " "
         - NAME@6..16
             - IDENT@6..16 "__typename"
-        - ERROR@16..19 "EOF"
+        - ERROR@16..16 ""
```

Comment:
Not sure what to make of this here, the `OPERATION_DEFINITION` is now showing the correct character span (the source is 16 characters long) but I don't know why this error message is empty.

Reply:
It's an error token, not an error message, so IMO it's correct like this